lib/llm/src/http/service/metrics.rs

```diff
@@ -80,6 +90,10 @@ impl Metrics {
     /// - `{prefix}_http_service_requests_total` - IntCounterVec for the total number of requests processed
     /// - `{prefix}_http_service_inflight_requests` - IntGaugeVec for the number of inflight requests
     /// - `{prefix}_http_service_request_duration_seconds` - HistogramVec for the duration of requests
+    /// - `{prefix}_http_service_input_sequence_length` - HistogramVec for input sequence length in tokens
+    /// - `{prefix}_http_service_output_sequence_length` - HistogramVec for output sequence length in tokens
```

Comment:
Strictly, those should end with `_tokens` not `_length` because that's the unit:
https://prometheus.io/docs/practices/naming/